app/src/main/java/com/gcect/gcectapp/ui/fragments/HomeFragment.kt

```diff
@@ -228,6 +240,55 @@ class HomeFragment : Fragment(), OnHomeNoticeItemClickListener, OnBottomSheetIte
                     pd.cancel()
                 }
             })
+
+    }
+
+    fun createNotificationChannel() {
```

Comment:
Creating an extension function will be better choice , So we can reuse this in other fragments also

Reply:
Now It's also used in other fragment as I checked.

Reply:
never seen this kind of method in an extension function before, we can still use it in different fragments using the class instance?

usually, I've used the extension function for some view utility or activity or some internal class which I don't have access

Reply:
Please feel free to share your thoughts on this

Reply:
Yaa, I also used this common method on utils but for now according to this issue I have only implement in one place there is an another issue in which I have to implement that functionality where it is required with common method.
This is only for testing the functionality that's it's working.